bitnami/cert-manager/Chart.yaml

```
@@ -26,4 +26,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/cert-manager-webhook
   - https://github.com/bitnami/containers/tree/main/bitnami/cainjector
   - https://github.com/jetstack/cert-manager
-version: 0.8.12
+version: 0.8.13
```

Thanks for your contribution! This is a new feature, so it should be a minor version. Can you update this, please?
```suggestion
version: 0.9.0
```